Assets/Mirror/Tests/Editor/Generated/CollectionWriterTests.gen.cs

```diff
@@ -10,7 +10,7 @@ namespace Mirror.Tests.Generated.CollectionWriters
 
     public class Array_int_Test
     {
-        class Message : MessageBase
+        public class Message : MessageBase
```

Comment:
this class is generated by `CollectionWriterGenerator`, CollectionWriterGenerator should be updated and run to rather than changing this file directly

Reply:
ohhh,  ok,  will fix,  tx